packages/core-contracts/contracts/utils/ERC20.sol

```diff
@@ -0,0 +1,101 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../interfaces/IERC20.sol";
+
+abstract contract ERC20 is IERC20 {
```

Comment:
Should this contract live in a `token` folder instead of `utils`?

Reply:
yeap, I was wondering about the naming, utils was the easy choice.